maps/randomvaults/Laundromat.dmm

```diff
@@ -0,0 +1,1301 @@
+	},
+/turf/space,
+/area)
+"Jh" = (
+/obj/spacepod/taxi,
+/obj/structure/lattice,
+/turf/space,
+/area)
+"Js" = (
+/obj/machinery/portable_atmospherics/canister/nitrogen,
+/turf/simulated/floor{
+	icon_state = "dark vault full"
+	},
+/area/vault/laundromat)
+"Kj" = (
+/turf/simulated/floor{
+	icon_state = "whitecorner";
+	dir = 4
+	},
+/area/vault/laundromat)
+"KA" = (
+/obj/machinery/light/he{
+	dir = 8
+	},
+/obj/abstract/map/spawner/laundromat/clothing,
+/turf/simulated/floor{
+	icon_state = "dark vault full"
+	},
+/area/vault/laundromat)
+"Lk" = (
+/obj/item/stack/sheet/mineral/plasma{
+	amount = 30
+	},
+/obj/item/stack/sheet/mineral/plasma{
+	amount = 30
+	},
+/obj/structure/rack,
+/turf/simulated/floor{
+	icon_state = "dark vault full"
+	},
+/area/vault/laundromat)
+"LA" = (
+/turf/simulated/floor{
+	dir = 1;
+	icon_state = "whitehall"
+	},
+/area/vault/laundromat)
+"LG" = (
+/obj/structure/grille,
+/obj/structure/window/full/reinforced,
+/obj/structure/window/reinforced{
+	dir = 4
+	},
+/obj/structure/window/reinforced{
+	dir = 8
+	},
+/obj/structure/window/reinforced{
+	dir = 1
+	},
+/turf/simulated/floor/plating,
+/area/vault/laundromat)
+"LJ" = (
+/obj/machinery/light/he,
+/turf/simulated/floor{
+	icon_state = "dark vault full"
+	},
+/area/vault/laundromat)
+"LX" = (
+/obj/effect/landmark/corpse/captain,
+/obj/item/weapon/stool,
+/turf/simulated/floor/glass,
+/area/vault/laundromat)
+"Mh" = (
+/obj/abstract/map/spawner/laundromat/clothing,
+/turf/simulated/floor,
+/area/vault/laundromat)
+"Ng" = (
+/obj/structure/rack,
+/obj/item/weapon/tank/air,
+/obj/item/weapon/tank/air,
+/obj/item/weapon/tank/nitrogen,
+/obj/item/weapon/tank/plasma/plasmaman,
+/obj/item/weapon/tank/oxygen,
+/turf/simulated/floor,
+/area/vault/laundromat)
+"Ny" = (
+/obj/effect/landmark/corpse/cargotechnician,
+/turf/simulated/floor,
+/area/vault/laundromat)
+"NN" = (
+/obj/machinery/computer/security/telescreen/entertainment/spesstv/flatscreen,
+/turf/simulated/wall,
+/area/vault/laundromat)
+"Of" = (
+/obj/effect/landmark/corpse,
+/turf/simulated/floor,
+/area/vault/laundromat)
+"Ov" = (
+/obj/structure/window,
+/turf/simulated/floor,
+/area/vault/laundromat)
+"Pt" = (
+/obj/structure/catwalk{
+	icon_state = "catwalk12"
+	},
+/obj/structure/railing{
+	dir = 1
+	},
+/turf/space,
+/area)
+"Qz" = (
+/obj/effect/landmark/corpse/vox,
+/obj/structure/lattice,
+/turf/space,
+/area)
+"QJ" = (
+/obj/structure/table/reinforced,
+/turf/simulated/floor{
+	icon_state = "dark vault full"
+	},
+/area/vault/laundromat)
+"QX" = (
+/obj/structure/window{
+	dir = 1
+	},
+/obj/structure/cable/yellow{
+	icon_state = "2-4"
+	},
+/turf/simulated/floor,
+/area/vault/laundromat)
+"Rr" = (
+/obj/machinery/door/airlock/external{
+	icon = 'icons/obj/doors/Doorglass.dmi';
+	name = "Laundromat";
+	opacity = 0
+	},
+/turf/simulated/floor,
+/area/vault/laundromat)
+"RS" = (
+/turf/simulated/floor{
+	icon_state = "dark vault full"
+	},
+/area/vault/laundromat)
+"RU" = (
+/obj/structure/ladder{
```

Comment:
is this latter floating in space? it appears to be on a space tile

Reply:
also you should make the underground one have a lower height variable so that it knows which to make "up"

Reply:
So have the drug lab have height 0 and the laundromat height 1? would that work? Not sure how they work.

Reply:
Yes, but you set the var on the ladder itself

Reply:
fixed